mesonbuild/build.py

```diff
@@ -2083,6 +2072,9 @@ def post_init(self) -> None:
         # libfoo.a. However, we cannot use foo.lib because that's the same as
         # the import library. Using libfoo.a is ok because people using MSVC
         # always pass the library filename while linking anyway.
+        #
+        # See our FAQ for more detailed rational:
```

Comment:
s/rational/rationale

Reply:
TIL: 'rational' is an adjective and 'rationale' is a noun. Fixed.